main.js

```diff
@@ -41,15 +41,23 @@ function showIdea() {
   var newIdea = document.createElement("article");
   var newIdeaTitle = document.createElement("h2");
   var newIdeaBody = document.createElement("p");
+  var deleteContainer = document.createElement("section");
```

Comment:
@ben-rosner-williamsburg Can we actually rename this container to something that isn't quite so specific? This was the main thing I was thinking about when we were talking to Travis about updating naming and stuff. This little header-container-thingie houses both the delete AND fave buttons, so just calling it `deleteContainer` doesn't quite make sense.

Reply:
@ericahagle I renamed this to headerContainer. Let me know if that's better.

Reply:
@ben-rosner-williamsburg Would something like `ideaBtnContainer` or `newIdeaBtnContainer` be better because it's more specific? It's basically the container to house the two buttons. I don't think it serves any other purpose down the road.

Reply:
@ericahagle Got it. Will rename.

Reply:
@ericahagle Just pushed back up.

Reply:
Primo! Looks great, @ben-rosner-williamsburg. Thanks for updating that!